tox.ini

```diff
@@ -4,11 +4,11 @@ skipsdist = true
 
 [testenv:molecule]
 deps =
-  ansible
+  ansible~=4.10
   ansible-lint
   flake8
-  molecule-docker
-  molecule-podman
+  molecule
+  molecule-containers
   molecule-vagrant
```

Comment:
Can we drop vagrant too?

Reply:
Finally I will re-work those lines, I accidentally reverted Francois' changes